unittest/buildtool/container_commands_test.py

```diff
@@ -146,7 +166,8 @@ def test_tag_containers(self):
         mock_regctl_image_copy.assert_has_calls(calls)
 
         # Should only be eight permutations, no more (e.g: NOT monitoring-third-party)
-        self.assertEqual(mock_regctl_image_copy.call_count, 8)
+        # Should be 12 when tagging JRE 11 images for front50 only
```

Comment:
I maybe haven't stared at this enough...but why not mention igor since it's in `SPINNAKER_JAVA11_VARIANT_REPOSITORY_NAMES` ?

Reply:
The test only handles images in the test bom in the same folder (front50 and monitoring daemon)